dask_expr/_accessor.py

```diff
@@ -94,4 +94,5 @@ def operation(obj, accessor, attr, args, kwargs):
 
 class FunctionMapIndex(FunctionMap):
     def _divisions(self):
+        # TODO: We can do better here
```

Comment:
I've seen these todo's, kinda assuming you have a method for dealing with them or do we want to be strict about making follow up issues?

Reply:
grepping when we are happy with the state of the test suite,

this specific occurrence isn't super important, so I avoided creating an issue, it's a tiny annoyance for now